docs/docs/integrations/embedding-stores/index.md

```diff
@@ -13,6 +13,7 @@ sidebar_position: 0
 | [Azure CosmosDB NoSQL](/integrations/embedding-stores/azure-cosmos-nosql)             | ✅                |                            |                     |
 | [Cassandra](/integrations/embedding-stores/cassandra)                                 | ✅                |                            |                     |
 | [Chroma](/integrations/embedding-stores/chroma)                                       | ✅                | ✅                          | ✅                   |
+| [Coherence](/integrations/embedding-stores/coherence)                          | ✅                | ✅                     | ✅                   |
```

Comment:
```suggestion
| [Oracle Coherence](/integrations/embedding-stores/oracle-coherence)                          | ✅                | ✅                     | ✅                   |
```

Reply:
What about naming it Oracle Coherence?
This will require moving it below close to Oracle

Reply:
I'd prefer not to -- it's too long, and all of our artifacts are just `coherence*`, not `oracle-coherence`, so leaving it the way it is now feels more natural.